src/lib/dialog/dialog-config.ts

```diff
@@ -74,7 +74,7 @@ export class MatDialogConfig<D = any> {
   maxWidth?: number | string = '80vw';
 
   /** Max-height of the dialog. If a number is provided, pixel units are assumed. */
-  maxHeight?: number | string;
+  maxHeight?: number | string = '80vh';
```

Comment:
Was this `65vh` before? If so, we probably would need to keep the same max-height until a major version change

Reply:
It wasn't, it was the scrollable content that was `65vh`, this applies the height to the overlay pane itself.